server/graphql/v2/interface/Account.js

```diff
@@ -780,10 +803,11 @@ export const AccountFields = {
           'Whether to include expired payment methods. Payment methods expired since more than 6 months will never be returned.',
       },
     },
-    description: 'The list of payment methods that this collective can use to pay for Orders',
+    description:
+      'The list of payment methods that this collective can use to pay for Orders. Admin only. Scope: "orders".',
     async resolve(collective, args, req) {
-      if (!req.remoteUser?.isAdminOfCollective(collective)) {
-        return [];
+      if (!req.remoteUser?.isAdminOfCollective(collective) || !checkScope(req, 'paymentMethods')) {
+        return null;
```

Comment:
Returning `null` while `type` says `GraphQLNonNull` will generate an ugly error. It sounds like the right thing to do though, so I'd remove the `GraphQLNonNull`.